code/modules/xenomorph/xeno_structures.dm

```diff
@@ -2,6 +2,18 @@
 	hit_sound = "alien_resin_break"
 	layer = RESIN_STRUCTURE_LAYER
 	resistance_flags = UNACIDABLE
+	///If killing the silo should not alert the hive
+	var/silent = FALSE
+
+/obj/structure/resin/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_DROPSHIP_HIJACKED, .proc/destroy_on_hijack)
+
+///Signal handler to delete the resin structure when the alamo is hijacked
+/obj/structure/resin/proc/destroy_on_hijack()
```

Comment:
resin structures are:
xeno turrets
silos
and resin pod

Resin pod being qdeleted on hijack is whatever IMO